base/src/main/java/com/epmedu/animeal/base/theme/Color.kt

```diff
@@ -21,4 +21,7 @@ internal object AnimealColor {
 object CustomColor {
     val Facebook = Color(0xFF438BED)
     val Google = Color(0xFF000000)
+    val LynxWhite = Color(0xFFF7F7F7)
+    val DarkerGrey = Color(0xFF99A8B0)
+    val CursorColor = Color(0xFF000000)
```

Comment:
Now you can hardcode color for the cursor since we agreed on this on the last meeting)